code/__HELPERS/icons.dm

```diff
@@ -872,6 +872,52 @@ The _flatIcons list is a cache for generated icon files.
 	del(atom_icon)
 	return text_image
 
+// Pick a random animal instead of the icon, and use that instead
+/proc/getRandomAnimalImage(atom/A)
+	var/list/icons = list()
+	var/icon/selected
+
+	icons += new/icon('icons/mob/animal.dmi', "cow")

```

Comment:
I don't know if there's any programmatic way of picking just the icons that I want without just listing them. The bolt of change code is similar, it just has a bunch of different mobs that you can become.